packages/fxa-payments-server/src/components/SubscriptionTitle/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import './index.scss';
+
+type SubscriptionTitleProps = {
+  screenType: string;
```

Comment:
Would be nice to have a sum type for this.  This type is telling me that I can pass in any string, which I don't think is true.